src/gwt/src/org/rstudio/studio/client/workbench/views/environment/view/EnvironmentObjectList.css

```diff
@@ -77,7 +77,7 @@ td.resizeCol {
 td.valueCol
 {
    border: 1px solid #f0f0f0;
-   white-space: nowrap;
+   white-space: pre;
```

Comment:
This is necessary so that output from `pandas.to_string()` is formatted correctly (as it uses whitespace for alignment)